TestingTravis/TestClass.swift

```diff
@@ -19,6 +19,10 @@ class TestClass: NSObject {
         //testing code owners
         //test 2
         let unused = 0
+        
+        let unused2 = 0
+        
```

Comment:
<table data-meta="generated_by_1">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/9b2dd97dbc9bd45e0af1d1a07dc0c52a9d753fc6/TestingTravis/TestClass.swift#L24"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TestClass.swift:24</code></td>
    </tr>
  </tbody>
</table>